### ISSUE TITLE
求助帖！Ask Help!

### ISSUE THREAD

**Issue by HustWolfzzb** [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning#2]
大哥，看你的名字，输中文应该没问题。
是这样的，小弟（我还在读大四，应该比你小吧~）做毕设也是选择的故障诊断方面的，但是目前决定采用决策树算法来基于数据驱动。现在苦于找不着数据来源。看你的这个Project，应该也是下了很多功夫的。不知道能否指点一下小弟，哪些地方能找到机械设备运行时候的大量数据的？如果哥你那儿有，能发我一份，那就更加感激不尽了！
------怕在那个问题里边你看不到，再单发一个，如有打扰，敬请见谅

**Comment by amazingdmz** [comment]
老哥我毕设也是这个题目。。

**Comment by CQUPanda** [comment]
你是研究生还是本科生？
照着现有文献做一遍就可以了




------------------ 原始邮件 ------------------
发件人: "memory-sb"<notifications@github.com>;
发送时间: 2018年4月17日(星期二) 中午12:13
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>; 
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning] 求助帖！Ask Help! (#2)




老哥我毕设也是这个题目。。

**Comment by Freezoner** [comment]
请问数据处理时，data文件夹里的是故障名.mat，一共10类，具体是怎么处理的？看image_matrix.m程序没太搞懂

**Comment by CQUPanda** [comment]
首先，给你提供了数据及文献，其次，我时间有限，答疑要收费的，谢谢



------------------ 原始邮件 ------------------
发件人: "Freezoner"; 
发送时间: 2018年4月18日(星期三) 上午10:36
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning";
抄送: "CQUPanda"<562251022@qq.com>;"Comment";
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning] 求助帖！Ask Help! (#2)




请问数据处理时，data文件夹里的是故障名.mat，一共10类，具体是怎么处理的？看image_matrix.m程序没太搞懂
 
&mdash;

**Comment by ZhangWei1993** [comment]
请大家参考我的这两篇论文：A New Deep Learning Model for Fault Diagnosis with Good Anti-Noise and Domain Adaptation Ability on Raw Vibration Signals与A deep convolutional neural network with new training methods for bearing fault diagnosis under noisy environment and different working load，更详细的请看我的硕士毕业论文：基于卷积神经网络的轴承故障诊断算法研究。若使用，请引用，谢谢。

**Comment by CQUPanda** [comment]
嗯


------------------ 原始邮件 ------------------
发件人: "Zhang Wei"<notifications@github.com>;
发送时间: 2018年4月18日(星期三) 上午10:47
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: "重庆大学机械学院孙攀"<562251022@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning] 求助帖！Ask Help! (#2)




请大家参考我的这两篇论文：A New Deep Learning Model for Fault Diagnosis with Good Anti-Noise and Domain Adaptation Ability on Raw Vibration Signals与A deep convolutional neural network with new training methods for bearing fault diagnosis under noisy environment and different working load，更详细的请看我的硕士毕业论文：基于卷积神经网络的轴承故障诊断算法研究。若使用，请引用，谢谢。

**Comment by zhangdinglin** [comment]
这位小哥 你有没有 直接能运行的 数据，我看看数据是什么样的，如果你要答疑收费 ，这都好说，请给我发邮件  339145761@qq.com 微信也是339145761 。我们详谈。

**Comment by chenglin1996** [comment]
您好，我已经把西储大学的轴承数据集已经下载了，但是运行MATLAB程序一直运行不同，一直都是下面这个出错误：
错误使用 load
无法读取文件 'D:\360安全浏览器下载\tensorflow所有程序\Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master\image_matrix1': 没有此类文件或目录。

您能帮我看看是怎么回事吗？麻烦您了，期待您的回复

**Comment by yaseralmatheel** [comment]
@chenglin1996 Did you find a solution for this problem?

**Comment by chenglin1996** [comment]
Yes, I have solved it. I read your paper and you said LSTM can be used. Could you give me some hints? May I add your WeChat or other contact information? I hope to learn from you.


发自我的iPhone

------------------ 原始邮件 ------------------
发件人: yaseralmatheel <notifications@github.com>
发送时间: 2019年7月17日 15:40
收件人: ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning <Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>
抄送: chenglin1996 <1834645316@qq.com>, Mention <mention@noreply.github.com>
主题: 回复：[ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning] 求助帖！Ask Help! (#2)




@chenglin1996 Did you find a solution for this problem?

**Comment by chenglin1996** [comment]
Yes, I have solved it. I read your paper and you said LSTM can be used. Could  you give me some hints? May I add your WeChat or other contact information? I  hope to learn from you.



------------------ 原始邮件 ------------------
发件人: "yaseralmatheel"<notifications@github.com>;
发送时间: 2019年7月17日(星期三) 下午3:40
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: "Leo"<1834645316@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning] 求助帖！Ask Help! (#2)




@chenglin1996 Did you find a solution for this problem?

**Comment by yaseralmatheel** [comment]
Hello,  actually it's not me who did LSMT.. MY Wechat number is 13087526205, could you please help me to send the modified data (training data and  training label  also test data and test label).  I just want to know how modify them.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by chunhuasong** [comment]
> 您好，我已经把西储大学的轴承数据集已经下载了，但是运行MATLAB程序一直运行不同，一直都是下面这个出错误：
> 错误使用 load
> 无法读取文件 'D:\360安全浏览器下载\tensorflow所有程序\Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master\image_matrix1': 没有此类文件或目录。
> 您能帮我看看是怎么回事吗？麻烦您了，期待您的回复
请问您怎么解决的这个问题？我也遇到一样的情况。期待您的回答
